book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+# Book class
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] @Al-Amin-Khan-Shakil please do the same, what I mentioned in my above comment related to `rentals`.